package.json

```
@@ -38,8 +38,8 @@
     "ts-react-toolbox": "^0.2.2"
   },
   "peerDependencies": {
-    "react": "^16.8.0",
-    "@types/react": "^16.8.0"
+    "react": ">16.8.0",
+    "@types/react": ">16.8.0"
```

Version 16.8.0 should still be allowed, so a `=` should be added.

```suggestion
    "react": ">=16.8.0",
    "@types/react": ">=16.8.0"
```

---

Another option would be to do the following instead (like is done [here](https://github.com/theKashey/react-focus-lock/blob/6117039973dbceb2af79f31ba74c2dedcf065a10/package.json#L50-L52)).

```suggestion
    "react": "^16.8.0 || ^17.0.0",
    "@types/react": "^16.8.0 || ^17.0.0"
```

---

love it 👍